lab_3_generate_by_ngrams/main.py

```diff
@@ -23,6 +23,8 @@ def __init__(self, end_of_word_token: str) -> None:
         Args:
             end_of_word_token (str): A token denoting word boundary
         """
+        self._end_of_word_token = end_of_word_token
+        self._storage = {"_": 0}
```

Comment:
Strange constant

Reply:
fixed

Reply:
no

Reply:
I fixed it locally before pushing, now the code is different